components/lib/mention/Mention.js

```diff
@@ -218,7 +220,8 @@ export const Mention = React.memo(
                 const prevText = value.substring(0, triggerState.index);
                 const nextText = value.substring(spaceIndex > -1 ? selectionStart : triggerState.index + currentText.length);
 
-                inputRef.current.value = `${prevText}${selectedText} ${nextText}`;
+                inputRef.current.value = spaceIndex > -1 ? `${prevText}${selectedText}${nextText}` : `${prevText}${selectedText} ${nextText}`;
```

Comment:
This ternary operator looks like both cases print out the exact same string?

Reply:
one of them contains a space.

So if the mention is inserted in front of a space it wont append another one. Not sure if thats too much of a behaviour change...